pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -698,32 +698,46 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podContainerC
 		result.Fail(fmt.Errorf("unable to get resource configuration processing resize for pod %s", pod.Name))
 		return
 	}
+
+	currentPodMemoryConfig, err := pcm.GetPodCgroupConfig(pod, v1.ResourceMemory)
+	if err != nil {
```

Comment:
@tallclair Should we call m.updatePodSandboxResources even if we get error on pcm.GetPodCgroupConfig?